README.md

```diff
@@ -154,13 +160,16 @@ use {
 * `content_spec_column`: Can be set to `true` (default `false`) to use instead of the preview.
   It will only show the type and number of lines next to the first line of the entry.
 * `on_select`:
-  * `move_to_front`: if the entry should be set as the lastest in the list when pressing the key to select a yank.
+  * `move_to_front`: if the entry should be set to last in the list when pressing the key to select a yank.
+  * `close_telescope`: if telescope should close whenever an item is selected.
 * `on_paste`:
   * `set_reg`: if the register should be populated when pressing the key to paste directly.
-  * `move_to_front`: if the entry should be set as the lastest in the list when pressing the key to paste directly.
+  * `move_to_front`: if the entry should be set to last in the list when pressing the key to paste directly.
+  * `close_telescope`: if telescope should close whenever a yank is pasted
 * `on_replay`:
   * `set_reg`: if the register should be populated when pressing the key to replay a recorded macro.
-  * `move_to_front`: if the entry should be set as the lastest in the list when pressing the key to replay a recorded macro.
+  * `move_to_front`: if the entry should be set to last in the list when pressing the key to replay a recorded macro.
```

Comment:
I noticed the `move_to_front` sections of the README had some grammatical errors (I think I was sleepy when I wrote that). I took the liberty to fix them

Reply:
Great, I missed this as well :)